openedx/core/djangoapps/plugins/plugin_contexts.py

```diff
@@ -0,0 +1,81 @@
+from importlib import import_module
+
+from logging import getLogger
+
+from openedx.core.lib.cache_utils import process_cached
+
+from . import constants, registry
+
+
+log = getLogger(__name__)
+
+def get_plugins_view_context(project_type, view_name, existing_context={}):
+    """
+    Returns a dict of additonal view context. Will check if any plugin apps
+    have that view in their view_context_config, and if so will call their
+    selected function to get their context dicts.
+    """
+    aggregate_context = {"plugins": {}}
+
+    # This functionality is cached
```

Comment:
Nit: you can rename the function so it's clear that it is cached, rather than have readers read an extra line of comment.  For example: `_get_cached_context_functions_for_view`.